src/main/java/uk/gov/hmcts/reform/translate/model/Dictionary.java

```diff
@@ -1,12 +1,18 @@
 package uk.gov.hmcts.reform.translate.model;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
 
 import java.util.Map;
 
-@AllArgsConstructor
-@Data
+@Getter
 public class Dictionary {
-    private Map<String, String> translations;
+
+    private final Map<String, String> translations;
+
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public Dictionary(@JsonProperty("translations") Map<String, String> translations) {
+        this.translations = translations;
+    }
```

Comment:
Do we need all these changes? The class looked a bit cleaner before, but fine if we do!

Reply:
yes we need for Jackson ro marshall and unmarshall the map     @JsonCreator duuu